qoqo_myqlm/interface/myqlm_interface.py

```diff
@@ -33,7 +35,7 @@ def myqlm_call_circuit(
     Returns:
         qlm.Program: translated circuit
     """
-    myqlm_program = qlm.Program()
+    myqlm_program = qlm.Program(default_gate_set=False)
```

Comment:
Could you add a comment as to why this has been added?

Reply:
this I thought about it, and probably is irrelevant in this case, what it does as defautl is to add a default gate set lookup table. I set it to false to save on this lookup table to not be added just to save few bits but is probably irrelevante. So I just went back to the previous case where I leave it unspecified (= True)